Source/carthage/CopyFrameworks.swift

```diff
@@ -27,11 +27,19 @@ public struct CopyFrameworksCommand: CommandType {
 
 				return combineLatest(SignalProducer(result: source), SignalProducer(result: target), SignalProducer(result: validArchitectures()))
 					.flatMap(.Merge) { (source, target, validArchitectures) -> SignalProducer<(), CarthageError> in
-						let copyFrameworks = copyFramework(source, target: target, validArchitectures: validArchitectures)
-						let copydSYMs = copyDebugSymbolsForFramework(source, validArchitectures: validArchitectures)
-						return combineLatest(copyFrameworks, copydSYMs)
-							.then(.empty)
-					}
+						return shouldIgnoreFramework(source, validArchitectures: validArchitectures)
+							.flatMap(.Concat) { shouldIgnore -> SignalProducer<(), CarthageError> in
+								if shouldIgnore {
+									carthage.println("warning: Ignoring \(frameworkName) because it does not support the current architecture\n")
```

Comment:
This should really have the bullets prefix that we use almost everywhere else. Can you add it and removing the `warning: ` prefix?